wp-includes/widgets/class-wp-widget-media.php

```diff
@@ -235,6 +244,36 @@ class="media-widget-instance-property"
 	}
 
 	/**
+	 * Filters the default media display states for items in the Media list table.
+	 *
+	 * @since 4.8.0
+	 * @access public
+	 *
+	 * @param array   $states An array of media states.
+	 * @param WP_Post $post   The current attachment object.
+	 * @return array
+	 */
+	public function display_media_state( $states, $post ) {
+
+		// Count how many times this attachment is used in widgets.
+		$use_count = 0;
+		foreach ( $this->get_settings() as $instance ) {
+			if ( isset( $instance['attachment_id'] ) && $instance['attachment_id'] === $post->ID ) {
+				$use_count++;
+			}
+		}
+
+		if ( $use_count > 1 ) {
+			/* translators: %d is widget count */
+			$states[] = sprintf( __( 'Media Widgets (%d)' ), $use_count );
```

Comment:
The plural conditional was an enhancement I came up with. Maybe it wouldn't be useful in practice, but I can imagine that it would be helpful for users to know how many widgets are using the image.

If we have both, then perhaps we'd need `media_library_state_singular_widget` and `media_library_state_multiple_widgets`